mobile.css

```diff
@@ -1,9 +1,7 @@
 
 /* Mobile responsiveness */
 
-html {
-  font-size: 62.5%;
-}
+
 
```

Comment:
- [ ] As mentioned by the previous reviewer and for the project requirements, kindly make sure your navigation menu options are not visible in a mobile screen. To handle that, you should use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px). In screen size < 768px only the Hamburger shoulb visible and the one >= 768px then your navigation options should visible and the Hamburger hidden. For a short remind about media queries, just refer this **[article](https://www.w3schools.com/css/css_rwd_mediaqueries.asp)**. 
### Your Design in mobile size

![mob](https://user-images.githubusercontent.com/83042742/230518664-b4dad258-bf1b-4a1c-88ee-b4fae3fdb0a9.gif)